src/components/dashboard/WalletActivity.tsx

```diff
@@ -45,28 +46,29 @@ export const WalletActivityComponent = ({
   const { t } = useTranslation();
 
   return (
-    <div>
-      <Title>{t("transactions")}</Title>
-      <br />
+    <div style={{ marginTop: "10px" }}>
       <Tabs
         animate={false}
         id="WalletTabs"
         renderActiveTabPanelOnly={true}
         vertical={false}
       >
         <Tab
-          id="all"
-          key="all"
-          title={t("all")}
-          style={{ fontSize: "16px" }}
+          id="in_progress"
```

Comment:
I think most people would much rather see their full transaction history by default on the left.

Reply:
I will try to add an indicator next to "In Progress" later, for now, I moved back the "All" tab to the beginning.